lib/Redis.php

```diff
@@ -771,7 +779,7 @@ public function hScan($key, $cursor, $pattern = null, $count = null): Promise
      * @return Promise
      * @yield bool
      */
-    public function hSet($key, $field, $value, $notExistingOnly = false): Promise
```

Comment:
This should probably be `$existingOnly = true`, but it's a pretty bad BC break, so let's leave it as is.